supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -23,12 +23,15 @@ case class JesBackendLifecycleActorFactory(configurationDescriptor: BackendConfi
   val jesConfiguration = new JesConfiguration(configurationDescriptor)
 
   override def workflowInitializationActorProps(workflowDescriptor: BackendWorkflowDescriptor,
-                                                calls: Seq[Call]): Option[Props] = {
-    Option(JesInitializationActor.props(workflowDescriptor, calls, jesConfiguration).withDispatcher("akka.dispatchers.backend-dispatcher"))
+                                                calls: Seq[Call],
+                                                serviceRegistryActor: ActorRef): Option[Props] = {
+    Option(JesInitializationActor.props(workflowDescriptor, calls, jesConfiguration, serviceRegistryActor).withDispatcher("akka.dispatchers.backend-dispatcher"))
```

Comment:
I can put it in, this PR already is a touch kitchen sinky anyways, what's another little thing